### ISSUE TITLE
Old hub banner is still accessible

### ISSUE THREAD

**Issue by temi** [AtlasOfLivingAustralia/biocollect#992]
User can still access old banner by going to this [link](https://biocollect.ala.org.au/project/projectFinder). This is not desirable sometimes since the images are different.

for example 
[test server's landing page](https://biocollect-test.ala.org.au/ala)
[test server's project finder](https://biocollect-test.ala.org.au/project/projectFinder)

**Comment by matilda2812** [comment]
@temi is this still an issue?

**Comment by pbrenton** [comment]
Yes, all publicaly accessible sites should be updated to reflect current and consistent branding. Also need to change the "powered by" icon in the hubs footer to "Powered by BioCollect a product of the Atlas of Living Australia".
